src/Polly.Testing/ResilienceStrategyType.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace Polly.Testing;
+
+/// <summary>
+/// The type of resilience strategy.
+/// </summary>
+public enum ResilienceStrategyType
```

Comment:
As per previous comment, I think having this as an enum makes things a bit too tightly bound and hard to extend, particularly with it not actually depending on some of the strategies directly but then directly exposing concepts related to them.

Reply:
Dropped the enum.